Cesium3DTilesSelection/src/CesiumIonTilesetLoader.h

```diff
@@ -0,0 +1,56 @@
+#pragma once
+
+#include "TilesetContentLoaderResult.h"
+
+#include <Cesium3DTilesSelection/TilesetContentLoader.h>
+#include <Cesium3DTilesSelection/TilesetExternals.h>
+
+#include <functional>
+#include <string>
+
+namespace Cesium3DTilesSelection {
+class CesiumIonTilesetLoader : public TilesetContentLoader {
+  enum class TokenRefreshState { None, Loading, Done, Failed };
+
+public:
+  using AuthorizationHeaderChangeListener = std::function<
+      void(const std::string& header, const std::string& headerValue)>;
+
+  CesiumIonTilesetLoader(
+      uint32_t ionAssetID,
+      std::string&& ionAccessToken,
```

Comment:
because we do capture some of that strings temporarily in the lambda in `CesiumIonTilesetLoader::createLoader()`, so I opt to use move for the constructor. It can save a little bit of allocations, so not much of performance advantage